index.js

```diff
@@ -25,16 +25,37 @@ function titleSlug(title) {
     return title.trim().replace(/\W/g, '_').substring(0, 251);
 }
 
-function appendImageUrlToErrorStack(imageObject, err) {
-    var output = (imageObject.imageUrl) ?
-        '\nnemo-screenshot\n' + imageObject.imageUrl + '\n' :
-        '\nnemo-screenshot::' + JSON.stringify(imageObject) + '::nemo-screenshot';
+function appendImageUrlToStackTrace(imageObject, err) {
+    var output;
+
+    if (imageObject.imageUrl) {
+        output = '\nnemo-screenshot (workspace): ' + imageObject.imageUrl + '\n' +
+                 'nemo-screenshot (archived): ' + imageObject.archivedImageUrl + '\n';
+    } else {
+        output = '\nnemo-screenshot::' + JSON.stringify(imageObject) + '::nemo-screenshot';
+    }
 
     if (err) {
         err.stack = err.stack + output;
     }
 }
 
+function formatJenkinsImageUrls(screenShotPath, imageName) {
+    var jenkinsUrl = process.env.JENKINS_URL,
+        buildUrl = process.env.BUILD_URL,
+        jobName = process.env.JOB_NAME,
+        workspace = process.env.WORKSPACE,
+        relImagePath = screenShotPath.substr(workspace.length);
```

Comment:
@nikulkarni Yeah, we can't be absolutely sure for the variables, added the check. Thanks!